be1-go/channel/election/mod.go

```diff
@@ -393,6 +410,25 @@ func (c *Channel) processElectionEnd(msg message.Message) error {
 	return nil
 }
 
+func (c *Channel) processElectionResult(msg message.Message, msgData interface{}) error {
```

Comment:
Is this for server to server communication ? It seems that beforehand, nothing was done in handleMessage for that

Reply:
Yes nothing was done before but I felt that I should at least verify that the message is somewhat correct even though the method doesn't do anything. It should maybe help detect errors in other part of the system.